src/App.tsx

```diff
@@ -15,35 +16,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortOption {
+  default = 'default',
```

Comment:
You don't need to make it unless you use it somewhere
```suggestion
  DEFAULT,
```